src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -385,6 +391,72 @@ export function MapAnalysisImpl(props: Props & CompleteAppState) {
     );
   };
 
+  /**
+   * The following code and styles are for demonstration purposes
+   * at this point. After #1671 is merged, we can implement these
+   * menu buttons and their associated panels for real.
+   */
+  const buttonStyles: React.CSSProperties = {
+    background: 'transparent',
+    borderColor: 'transparent',
+    fontSize: 16,
+    margin: 0,
+    padding: 0,
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+  };
+  const iconStyles: React.CSSProperties = {
+    height: 25,
+    width: 25,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  };
+  const labelStyles: React.CSSProperties = {
+    marginLeft: '0.5rem',
+  };
```

Comment:
Keeping these as CSS-in-JS because we've got to align on what we're doing. Moreover, these styles apply to navigation items and I think we need to talk about our expectations for those.

For example, when I've clicked on "Filter Data," is there an active state for the side navigation item while the draggable panel is open?

And the design has a big site name header at the top of it—do we really want that?

This PR a minimal first iteration at the side menu.